### PR TITLE
Add example scripts

### PR DESCRIPTION
**Pull request by AngeloCG97**
### What does this PR have?

- Add transaction scripts
- Add deploy contract scripts

### CODE REVIEW

File: deploy contract/holacontrato.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
+deploy_holacontrato_contracts_to_lacchain() {
```
Comment (leisterfrancisco) [inline deploy contract/holacontrato.sh]: deploy_holacontrato_contracts_to_lacchain -> deploy_holacontrato

File: deploy contract/holacontrato.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    jq -s '[.[].actions[]]' $TEMP_DIR/tx1.json $TEMP_DIR/tx2.json >$TEMP_DIR/tx3.json
+
+    echo '4. merge transaction'
+    jq '.actions = input' $TEMP_DIR/tx1.json $TEMP_DIR/tx3.json >$TEMP_DIR/tx4.json
+
+    echo '5. sign transaction'
+    cleos -u <EOS_API_URL> -r "Accept-Encoding: identity" push transaction $TEMP_DIR/tx4.json -p costarica@writer -p holacontrato@active
+}
+
+deploy_holacontrato_contracts_to_lacchain
```
Comment (leisterfrancisco) [inline deploy contract/holacontrato.sh]: same here

File: prepare transaction/Shell Script/holacontrato.sh [hunk 3]
```diff
@@ -0,0 +1,13 @@
+holacontrato_lacchain() {
```
Comment (leisterfrancisco) [inline prepare transaction/Shell Script/holacontrato.sh]: change function name for something more descriptive like llamar_holacontrato or ejecutar_hola something like that

File: prepare transaction/eosJS/holacontrato.js [hunk 4]
```diff
@@ -0,0 +1,45 @@
+
+const { eosioConfig } = require('config')
+
+const holacontrato_lacchain = ({
```
Comment (leisterfrancisco) [inline prepare transaction/eosJS/holacontrato.js]: same here, make descriptive name and representing an action

File: prepare transaction/eosJS/holacontrato.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
+
```
Comment (leisterfrancisco) [inline prepare transaction/eosJS/holacontrato.js]: remove empty line

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]

**Comment by xavier506** [comment]
@AngeloCG97  lets add a readme file based on the templete @leisterfrancisco  has been using

File: prepare transaction/Shell Script/holacontrato.sh [hunk 6]
```diff
@@ -0,0 +1,13 @@
+run_holacontrato() {
```
Comment (xavier506) [inline prepare transaction/Shell Script/holacontrato.sh]: @AngeloCG97  avoid using spaces in directory and file names

File: prepare transaction/eosJS/holacontrato.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+const { eosioConfig } = require('config')
```
Comment (xavier506) [inline prepare transaction/eosJS/holacontrato.js]: does. this script work? I think you need to import `eosjs`

Comment (xavier506) [reply prepare transaction/eosJS/holacontrato.js]: @AngeloCG97  please see these examples:

https://developers.eos.io/manuals/eosjs/latest/transaction-examples

File: prepareTransaction/eosJS/holacontrato.js [hunk 8]
```diff
@@ -0,0 +1,46 @@
+
+const eosUtil = require('./eos')
```
Comment (xavier506) [inline prepareTransaction/eosJS/holacontrato.js]: @AngeloCG97  Have a loook at this documentation

https://developers.eos.io/manuals/eosjs/latest/transaction-examples

**Pull request merged**